packages/types/src/options.ts

```diff
@@ -19,6 +19,7 @@ export interface FinderContext {
   name: string
   version: string
   readManifest: () => DependencyManifest
+  alias?: string
```

Comment:
I am not sure it will ever be undefined. It'll just equal the name in most cases.

Reply:
Indeed, I've updated.
[refactor: make alias required instead of optional](https://github.com/pnpm/pnpm/pull/10084/commits/af59daa7607c49c58cc2504830205b932fa31e37)